exceptions/RestfulCallbackException.php

```
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Contains RestfulCallbackException
+ */
+
+class RestfulCallbackException extends RestfulException {
```

This exception isn't needed, as the classes extending `RestfulException` should throw an exception that was caused due to a _client_ error, not internal error.